addon.xml

```diff
@@ -10,7 +10,7 @@
     <extension point="xbmc.service" library="service.py"/>
 
     <extension point="xbmc.addon.metadata">
-        <summary lang="en_GB">Skip YouTube in-video sponsors</summary>
+        <summary lang="en_GB">Skip YouTube in-video sponsors (Port)</summary>
```

Comment:
I think this reads somewhat poorly because the summary doesn't mention the name "SponsorBlock".